### PR TITLE
Rename display_name to name in nereid user

### PR DESCRIPTION
**Pull request by uzayr**
[#121927433]

### CODE REVIEW

File: trytond_nereid/user.py [hunk 1]
```diff
@@ -148,6 +153,32 @@ class NereidUser(ModelSQL, ModelView):
...
+#    def get_display_name(self, records, name):
+    def get_display_name(self, name):
+        "Returns the display name"
+        warnings.warn(
+            "Nereid display_name field is deprecated, "
+            "use name instead",
+            DeprecationWarning, stacklevel=2
+        )
+        return self.name
+#        res = {}
```
Comment (tarunbhardwaj) [inline trytond_nereid/user.py]: ?

File: trytond_nereid/user.py [hunk 2]
```diff
@@ -148,6 +153,32 @@ class NereidUser(ModelSQL, ModelView):
     email_verified = fields.Boolean("Email Verified")
     active = fields.Boolean('Active')
 
+#    def get_display_name(self, records, name):
```
Comment (tarunbhardwaj) [inline trytond_nereid/user.py]: ?

File: trytond_nereid/user.py [hunk 3]
```diff
@@ -148,6 +153,32 @@ class NereidUser(ModelSQL, ModelView):
...
+    def get_display_name(self, name):
+        "Returns the display name"
+        warnings.warn(
+            "Nereid display_name field is deprecated, "
+            "use name instead",
+            DeprecationWarning, stacklevel=2
+        )
+        return self.name
```
Comment (tarunbhardwaj) [inline trytond_nereid/user.py]: make it class method

File: trytond_nereid/user.py [hunk 4]
```diff
@@ -114,7 +115,11 @@ class NereidUser(ModelSQL, ModelView):
         ondelete='CASCADE', select=1
     )
 
-    display_name = fields.Char('Display Name', required=True)
+    name = fields.Char('Name')
+    display_name = fields.Function(
+        fields.Char("Name"),
+        getter="get_display_name", setter="set_display_name",
```
Comment (prakashpp) [inline trytond_nereid/user.py]: you also need searcher for this.

**Pull request closed without merging**